dc-up-d.sh

```diff
@@ -2,12 +2,38 @@
 
 unset PROFILE
 
+main=`docker-compose  version  --short | cut -d '.' -f 1`
+minor=`docker-compose  version  --short | cut -d '.' -f 2`
+current=`docker-compose  version  --short`
+
+echo 'Checking docker-compose version'
+if [[ $main -lt 1 ]]; then
+  echo "$current is not supported docker-compose version, please upgrade to minimal supported version:1.28"
+  exit 1
+elif [[ $main -eq 1 ]]; then
+  if [[ $minor -lt 28 ]]; then
+    echo "$current is not supported docker-compose version, please upgrade to minimal supported version:1.28"
+    exit 1
+  fi
+fi
+
+echo 'Supported docker-compose version'
+
+
 if [ $# -eq 0 ]
 then
     if [ -z $DD_PROFILE ]
     then
-        echo "No profile supplied"
-        exit 1
+        echo "No profile supplied, running default: mysql-rabbitmq"
+        PROFILE="mysql-rabbitmq"
+        echo "Other supported profiles:
+          mysql-rabbitmq*
+          mysql-redis
+          postgres-rabbitmq
+          postgres-redis
+
+        Usage example: ./dc-up-d.sh mysql-redis
+        "
```

Comment:
I suggest to also update the docs to have an example like `./dc-up-d.sh mysql-redis` (instead of assuming / relying on DD_PROFILE to be set).

Reply:
https://github.com/DefectDojo/django-DefectDojo/pull/6165/files introduced some examples in the docs.

Reply:
ok, didn't notice that. still my suggestion would to make the first command in each line contain the profile name as well and mention the DD_PROFILE thing only after the examples

Reply:
Added into doc, and did small changes in order.